README.md

```diff
@@ -13,13 +13,13 @@ The `apps:free` Heroku CLI plugin shows the apps using free dynos or free data (
 To get started, install the plugin from NPM.
 
 ```
-$ heroku plugins:install @heroku-cli/plugin-free
+heroku plugins:install @heroku-cli/plugin-free
 ```
 
 With the plugin installed, you can run it without any arguments to fetch all apps across personal and any Heroku Team you’re a part of.
 
 ```
-$ heroku apps:free
```

Comment:
This one should stay be the previous can be removed. Please update to add the $ here back.

Reply:
Done